packages/contentstack-config/src/interfaces/index.ts

```diff
@@ -14,5 +14,5 @@ export interface Region {
   name: string;
   cma: string;
   cda: string;
-  uiHost: string;
+  uiHost?: string;
```

Comment:
@shafeeqd959 , uiHost is a required parameter. right ? 
https://www.contentstack.com/docs/developers/cli/configure-regions-in-the-cli

Reply:
Done, thanks